Source/WebSocket.swift

```diff
@@ -355,6 +355,9 @@ open class WebSocket : NSObject, StreamDelegate {
                         return
                     }
                 }
+            } else {
```

Comment:
Hi, thanks for the nice response. I'm a bit confused, because I didn't realise that it's possible to connect to an encrypted socket without ciphers (it doesn't work on the server I'm testing on). Do you have a suggestion what condition to test for, so that the error message is only shown when the connection actually fails?